packages/connect-playwright/src/testdata/gen/test_pb.ts

```diff
@@ -18,21 +18,14 @@
 
 import type { GenFile, GenService } from "@bufbuild/protobuf/codegenv1";
 import { fileDesc, serviceDesc } from "@bufbuild/protobuf/codegenv1";
-import type {
-  Int32ValueSchema,
-  StringValueSchema,
-} from "@bufbuild/protobuf/wkt";
+import type { Int32ValueSchema, StringValueSchema } from "@bufbuild/protobuf/wkt";
```

Comment:
Looks like "generate" and "format" are racing here. We should ignore src/testdata in "format" in packages/connect-playwright/package.json.